src/io/flutter/view/DebugDrawAction.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018 The Chromium Authors. All rights reserved.
+ * Use of this source code is governed by a BSD-style license that can be
+ * found in the LICENSE file.
+ */
+package io.flutter.view;
+
+import com.intellij.icons.AllIcons;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import io.flutter.FlutterBundle;
+import io.flutter.run.daemon.FlutterApp;
+import org.jetbrains.annotations.NotNull;
+
+class DebugDrawAction extends FlutterViewToggleableAction {
+  DebugDrawAction(@NotNull FlutterApp app) {
+    super(app, FlutterBundle.message("flutter.view.debugPaint.text"), FlutterBundle.message("flutter.view.debugPaint.description"),
+          AllIcons.General.TbShown);
+
+    setExtensionCommand("ext.flutter.debugPaint");
+  }
+
+  @Override
+  protected void perform(AnActionEvent event) {
```

Comment:
in a follow up cl it would be nice to get rid of all these redundant perform methods that are doing the same thing.

Reply:
Doing this in a follow up CL.